std_msgs/Header header

VertexMsg[] vertex
SupportVertexMsg[] support_vertex
EdgeMsg[] edge
float32 threshold
float32 offset_angle